### DISCUSSION TITLE
Query

### DISCUSSION THREAD

**Discussion by Thre4dripper** [ossamamehmood/Hacktoberfest#1758] [Q&A] [answered]
Hi,
What is the criteria of participating in hacktoberfest as a maintainer of repo.

**Comment by ossamamehmood** [comment, marked as answer]
MAINTAINERS

Prepare your project for contributions by following these best practices:

Add the “hacktoberfest” topic to your repository to OPT-IN TO HACKTOBERFEST and indicate you’re looking for contributions.

Apply the “hacktoberfest” label to issues you want contributors to help with in your GitHub or GitLab project.

Add a CONTRIBUTING.md file with contribution guidelines to your repository.

Choose issues that have a well-defined scope and are self-contained.

Adopt a code of conduct to create a greater sense of inclusion and community for contributors.

Be ready to review pull/merge requests, accepting those that are valid by merging them, leaving an overall approving review, or by adding the “hacktoberfest-accepted” label.

Reject any spammy requests you receive by labeling them as “spam,” and any other invalid contributions by closing them or labeling them as “invalid.”

REWARD FOR MAINTAINERS

The hard work of our Maintainers is the reason Hacktoberfest exists, so we want you to have the opportunity to receive your very own Hacktoberfest 2022 reward. In order to become eligible for the opportunity to win a Reward Kit all a Maintainer will have to do is complete four or more maintainer actions on unique PR/MRs in repos participating in Hacktoberfest. Here are the Maintainer actions that qualify:

Merge unique PR/MRs

Provide an approving review of a PR/MR

Add the “hacktoberfest-accepted” label

Add any label with the word “invalid” or “spam”

*REWARD KITS ARE LIMITED; WE WILL REACH OUT TO QUALIFYING MAINTAINERS WHO ARE SELECTED TO RECEIVE A REWARD KIT AFTER HACKTOBERFEST ENDS.
